### PR TITLE
update footer guidance

### PR DESCRIPTION
**Pull request by amyhupe**
This PR updates the footer guidance with information on standard footer links and who to contact with questions.

### CODE REVIEW

**Comment by jpw** [comment]
Error: nature-footer/HISTORY.md file must be updated
  
  - _check-publish-file.js:23 checkPublishFile
    [frontend-toolkits]/[@springernature]/frontend-package-manager/lib/js/_valid    ate/_check-publish-file.js:23:9
  
  - _check-changed-files.js:30 checkFiles
    [frontend-toolkits]/[@springernature]/frontend-package-manager/lib/js/_valid    ate/_check-changed-files.js:30:8
  
  - _check-publication.js:32 checkPublicationConditions
    [frontend-toolkits]/[@springernature]/frontend-package-manager/lib/js/_valid    ate/_check-publication.js:32:9
  
  - next_tick.js:68 process._tickCallback
    internal/process/next_tick.js:68:7
  
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! frontend-toolkits@ validate: `sn-package-validate "-n"`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the frontend-toolkits@ validate script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     /home/travis/.npm/_logs/2022-03-22T16_50_41_248Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! frontend-toolkits@ build:ci: `npm run lint && npm run test:ci && npm run validate -- -n`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the frontend-toolkits@ build:ci script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     /home/travis/.npm/_logs/2022-03-22T16_50_41_264Z-debug.log
The command "npm run bootstrap && npm run build:ci" exited with 1.
cache.2
store build cache

File: toolkits/nature/packages/nature-footer/package.json [hunk 1]
```diff
@@ -1,9 +1,9 @@
 {
     "name": "@springernature/nature-footer",
-    "version": "2.1.1",
+    "version": "4.0.3",
```
Comment (alexkilgour) [inline toolkits/nature/packages/nature-footer/package.json:3]: Why is this changing here? All the other changes are to the `global-corporate-footer`, this is why travis is complaining.

Comment (alexkilgour) [reply toolkits/nature/packages/nature-footer/package.json:3]: Looks like this needs to stay at `2.1.1` and the `global-corporate-footer` needs to be updated to `4.0.3`

Comment (sturobson) [reply toolkits/nature/packages/nature-footer/package.json:3]: Ah-hah!

Comment (sturobson) [reply toolkits/nature/packages/nature-footer/package.json]: now I've made these changes - I don't think it's 'right' for me to review it (the initial task). @alexkilgour / @jpw is this something you could take a look at for @amyhupe 😄

**Review by alexkilgour — APPROVED** [review]
👍

**Pull request merged**